eth/tracers/api_bor.go

```diff
@@ -56,7 +57,8 @@ func (api *API) traceBlock2(ctx context.Context, block *types.Block, config *Tra
 	if config != nil && config.Reexec != nil {
 		reexec = *config.Reexec
 	}
-	statedb, err := api.backend.StateAtBlock(ctx, parent, reexec, nil, true)
+	// TODO: discuss consequences of setting preferDisk false.
```

Comment:
@ferranbt would also love your thoughts on this change as well, which is due to the latest geth merge. Basically, if setting false to prefer disk flag looks good to go or not?

Reply:
I think it does not matter since it is only used when provided a pre-state reference and we are not doing it.